ros/openapi/openapi.json

```diff
@@ -0,0 +1,214 @@
+{
+    "openapi": "3.0.1",
+    "info": {
+        "title": "Ros-backend API",
+        "description": "Flask Backend API for Resource Optimization Service",
+        "contact": {},
+        "license": {
+            "name": "GPLv3",
+            "url": "https://www.gnu.org/licenses/gpl-3.0.en.html"
+        },
+        "version": "v0.1"
+    },
+    "servers": [
+        {
+            "url": "{{ PATH_PREFIX }}/{{ APP_NAME }}/v0"
+        }
+    ],
+    "paths": {
+        "/status": {
+            "get": {
+                "summary": "Health check liveness call",
+                "description": "A small JSON indicating the application is deployed. This serves as both the liveness and readiness call.",
+                "responses": {
+                    "200": {
+                        "description": "OK",
+                        "content": {
+                            "application/json": {
+                                "schema": {
+                                    "$ref": "#/components/schemas/Status"
+                                }
+                            }
+                        }
+                    }
+                }
+            }
+        },
+        "/systems": {
+            "get": {
+                "summary": "Show all systems from host inventory having performance profile",
+                "description": "Show all systems from host inventory having performance profile",
+                "operationId": "listSystems",
+                "parameters": [
+                    {
+                        "name": "limit",
+                        "in": "query",
+                        "description": "Limit for paging, set -1 to return all",
+                        "schema": {
+                            "type": "integer"
+                        }
+                    },
+                    {
+                        "name": "offset",
+                        "in": "query",
+                        "description": "Offset for paging",
+                        "schema": {
+                            "type": "integer"
+                        }
+                    },
+                    {
+                        "name": "sort",
```

Comment:
As per API design guidelines - https://platform-docs.cloud.paas.psi.redhat.com/rest/sorting.html, for sorting it is recommended to set `sort_by` query parameter. 
I don't want to block the PR as it's in initial state but thought to share for reference.

Reply:
Yeah sure, let's consider this for refactoring when openapi specs would change after #18, makes sense?

Reply:
since we do not have sorting/searching/filtering options as of now, eliminating this part.